README.rst

```diff
@@ -620,16 +620,17 @@ from being displayed on the changelist view of Subnet and IP admin.
 ``OPENWISP_CONTROLLER_SUBNET_DIVISION_TYPES``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-+--------------+------------------------------------------------------------------------------------------------+
-| **type**:    | ``tuple``                                                                                      |
-+--------------+------------------------------------------------------------------------------------------------+
-| **default**: | .. code-block:: python                                                                         |
-|              |                                                                                                |
-|              |    (                                                                                           |
-|              |       ('openwisp_controller.subnet_division.rule_types.vpn.VpnSubnetDivisionRuleType', 'VPN'), |
-|              |    )                                                                                           |
-|              |                                                                                                |
-+--------------+------------------------------------------------------------------------------------------------+
++--------------+---------------------------------------------------------------------------------------------------------+
+| **type**:    | ``tuple``                                                                                               |
++--------------+---------------------------------------------------------------------------------------------------------+
+| **default**: | .. code-block:: python                                                                                  |
+|              |                                                                                                         |
+|              |    (                                                                                                    |
+|              |       ('openwisp_controller.subnet_division.rule_types.vpn.VpnSubnetDivisionRuleType', 'VPN'),          |
+|              |       ('openwisp_controller.subnet_division.rule_types.device.DeviceSubnetDivisionRuleType', 'Device'), |
+|              |    )                                                                                                    |
+|              |                                                                                                         |
++--------------+---------------------------------------------------------------------------------------------------------+
 
 Available types for Subject Division Rule objects. For more information on how
```

Comment:
I think we need to explain how these two rules work, I think we should create subheadings for each, otherwise users won't know how to use these.

Reply:
It made more sense to me to add the description here: https://github.com/openwisp/openwisp-controller/tree/issues/522-device-subnet-division#1-vpn

I think only developers will read the documentation section of settings and not users. While tutorial will be read by everyone. It does affect the continuity of the tutorial a bit. I will move it to some other section if you like.

Reply:
ok, let's add a link to those from here please (and fix the missing final dot in this sentence).